ui/src/app/views/workspace-edit-underserved-population/component.tsx

```diff
@@ -0,0 +1,320 @@
+      }
+    ]
+  }
+];
+
+export class WorkspaceUnderservedPopulation extends
+    React.Component<{value: Array<UnderservedPopulationEnum>, onChange: Function},
+    {show: boolean, value: Array<UnderservedPopulationEnum>}> {
+  constructor(props: any) {
+    super(props);
+    this.state = {show: false, value: !props.value ? [] : props.value};
+  }
+
+  flipShow() {
+    this.setState({
+      show: !this.state.show
+    });
+  }
+
+  focusCategoryChange(subCategory, value) {
+    if (!value) {
+      const index = this.state.value.findIndex(item => item === subCategory.id);
+      this.state.value.splice(index , 1);
+    } else if (value) {
+      this.state.value.push(subCategory);
+    }
+    this.props.onChange(this.state.value);
+  }
+
+
+  render() {
+    return <div style={styles.section}>
+      <div style={{display: 'flex', flexDirection: 'row'}}>
+        <div>
+          <Clickable onClick={() => this.flipShow()}>
+            {!this.state.show &&
+            <ClrIcon shape='plus-circle' style={styles.icon}/>
+            }
+            {this.state.show &&
+            <ClrIcon shape='minus-circle' style={styles.icon}/>
+            }
+          </Clickable>
+        </div>
+      <div>
+        <div style={styles.header}>
+          Focus on an underserved population
+          <TooltipTrigger content='A primary mission of the All of Us Research Program is to include
+            populations that are medically underserved and/or historically underrepresented in
+            biomedical research or who, because of systematic social disadvantage, experience
+            disparities in health. As a way to understand how much research is being conducted on
+            these populations, All of Us requests that you mark all options for underserved
+            populations that will be included in your research.'>
+            <InfoIcon style={styles.infoIcon}/>
+          </TooltipTrigger>
+        </div>
+        <label style={styles.focusLabel}>
+          This research will focus on, or include findings on, distinguishing characteristics
+          related to one or more underserved populations
+        </label>
+      </div>
+    </div>
+      {this.state.show && <div style={{paddingTop: '1rem'}}>
+        <h5>Additional information for underserved population</h5>
+        <label>Select all that apply. Only options for underserved populations are provided.
+        </label>
+        <div>
+          {FocusCategories.map (({title, categories}) => {
+            return <div key={title}>
+              <h4 style={styles.title}>
+                {title}
+              </h4>
+              <div style={{display: 'flex', flexWrap: 'wrap'}}>
+                {
+                  categories.map(({id, label}) => {
+                    const isIncluded = fp.includes(id, this.state.value);
+                    return <div style={styles.row}>
+                      <input type='checkbox' style={styles.checkBox} checked={isIncluded}
```

Comment:
FYI, there's a `CheckBox` component that abstracts over the basic checkbox and unpacks the `{target: {value}}` structure in `onChange`